sdk/core/azure-core/inc/credentials/policy/policies.hpp

```diff
@@ -24,40 +24,33 @@ namespace Azure { namespace Core { namespace Credentials { namespace Policy {
     void operator=(BearerTokenAuthenticationPolicy const&) = delete;
 
   public:
-    BearerTokenAuthenticationPolicy(
-        std::shared_ptr<TokenCredential const> const& credential,
-        std::string const& scope)
-        : m_credential(credential)
+    explicit BearerTokenAuthenticationPolicy(
```

Comment:
Nit, This looks unrelated to the telemetry policy feature.  Easier to track history when commits are broken into unique bug/feature commits.

Reply:
Yes, it ended up being more than I realized. In the future, I'll be splitting into smaller PRs.

Reply:
Resolving.